internal/server/data/access_key_test.go

```diff
@@ -291,23 +291,23 @@ func TestListAccessKeys(t *testing.T) {
 		createIdentities(t, db, user, otherUser)
 
 		first := &models.AccessKey{
-			Name:       "first",
+			Name:       "alpha",
```

Comment:
I changed these names so that the existing tests sort correctly. I changed the insertion order below so that the tests would fail without this change.